app/src/components/Hand.vue

```diff
@@ -52,7 +56,10 @@ export default {
     isCardDisabled(card) {
       // every cards are disabled when it's not the players turn (for now)
       if (!this.isUserTurn) {
-        return true;
+        // if player is beeing looted, he can select a card
+        if (this.currentPhase === GamePhase.ResolvingLoot) {
+          return false;
+        } else return true;
```

Comment:
Variation of the classic `if (true) return true`
```suggestion
        return !this.currentPhase === GamePhase.ResolvingLoot;
```

Reply:
Between the isCardDisabled (ie. is not enabled), and the not isUserTurn higher up, I'd rather keep is as readable as possible. And we're going to add an if-else anyway when implementing some other cards.